modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -23,7 +23,8 @@ const getContentFromBody = (editor: Editor, args: GetContentArgs, body: HTMLElem
     content = Tools.trim(TrimHtml.trimExternal(editor.serializer, body.innerHTML));
   } else if (args.format === 'text') {
     // return empty string for text format when editor is empty to avoid bogus elements being returned in content
-    content = editor.dom.isEmpty(body) ? '' : Zwsp.trim(body.innerText || body.textContent);
+    content = Zwsp.trim(body.innerText || body.textContent);
```

Comment:
You can remove `textContent` here while you are at it. Firefox has implemented innerText a long time ago see: https://caniuse.com/innertext